signalfx-org-metrics/metrics.yaml

```diff
@@ -1,3 +1,255 @@
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceivedByToken
+
+sf.org.apm.grossSpanBytesReceived:
+  brief: The number of bytes SignalFx receives for a span after decompression
+    before filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a span after
+    decompression before filtering and throttling.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpanBytesReceived
+
+sf.org.apm.grossSpanBytesReceivedByToken:
+  brief: The number of bytes SignalFx receives for a specific access token
+    for a span after decompression before filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a specific access token
+    for a span after decompression before filtering and throttling.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpanBytesReceivedByToken
+
+sf.org.apm.numSpanBytesReceived:
+  brief: The number of bytes SignalFx receives for a span after decompression
+    after filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a span after
+    decompression after filtering and throttling.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpanBytesReceived
+
+sf.org.apm.numSpanBytesReceivedByToken:
+ brief: The number of bytes SignalFx receives for a specific access token
+    for a span after decompression after filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a specific access token
+    for a span after decompression after filtering and throttling.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpanBytesReceivedByToken
+
+sf.org.apm.numAddSpansCallsByToken:
+  brief: The number of calls to a SignalFx ingest endpoint for a specific access token.
+  description: 'The number of calls to a SignalFx ingest endpoint for a specific access token.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numAddSpansCallsByToken
+
+sf.org.apm.numAddSpansCalls:
+  brief: The number of calls to a SignalFx ingest endpoint.
+  description: 'The number of calls to a SignalFx ingest endpoint.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numAddSpansCalls
+
+sf.org.apm.invalidSpansReceived:
+  brief: 
+  description: 
+  metric_type: counter
+  title: sf.org.apm.invalidSpansReceived
+
+sf.org.apm.invalidSpansReceivedByToken:
+  brief: 
+  description: 
+  metric_type: counter
+  title: sf.org.apm.invalidSpansReceivedByToken
+
+sf.org.apm.numTracesReceived:
+  brief: 
+  description: 
+  metric_type: counter
+  title: sf.org.apm.numTracesReceived
+
+sf.org.apm.numTroubleshootingMetricSets10s:
+  brief: 
```

Comment:
The cardinality of Troubleshooting MetricSets in each 10-second window. Data resolution is 10 seconds. Metric type is a gauge.

Note that we also produce `sf.org.apm.numTroubleshootingMetricSets` (don't see it in the list) which is the cardinality of TMSes in each 1-minute window; data resolution is 1 minute, metric type is also a gauge.